classes/class3/homework/submit_casts/app.js

```diff
@@ -0,0 +1,36 @@
+var express = require('express');
+var app = express();
+
+var path = require('path')
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var index = require('./routes/index.js')
+var exphbs = require('express-handlebars');
+var PORT = process.env.PORT || 3000;
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.engine('handlebars', exphbs({defaultLayout: 'main'}));
+app.set('view engine', 'handlebars');
+
+app.get('/',function(req,res){
+	res.render("home",{'paths':['/casts','/casts/new','/casts/bymember/Adam Archer','/casts/delete/old']});
+});
+
+app.get('/casts', index.casts);
+app.get('/casts/new', index.newcast);
+app.get('/casts/bymember/*', index.bymember);
+app.get('/casts/delete/old', index.deleteold);
+
+app.get('/cats', index.casts);
```

Comment:
Unnecessary routes, but I guess this is what we actually asked for so... nice workaround?